contents/julia_basics.md

```diff
@@ -16,6 +16,63 @@ Imagine that Julia is a fancy feature-loaded car, such as a brand-new Tesla.
 We'll just explain to you how to "drive the car, park it, and how to navigate in traffic".
 If you want to know what "all the buttons in the steering wheel and dashboard do", this is not the resource you are looking for.
 
+## Development environments {#sec:ide}
```

Comment:
Isn't this supposed to be title cased? Like "Development Environments"?

Reply:
Fixed in 0f55817.